Sources/OTelCore/OTel+Configuration.swift

```diff
@@ -376,6 +383,61 @@ extension OTel.Configuration {
     }
 }
 
+extension OTel.Configuration.TracesConfiguration {
+    /// Selection of traces sampler.
+    public struct SamplerConfiguration: Sendable {
+        package enum Backing: String, Sendable {
+            case alwaysOn = "always_on"
+            case alwaysOff = "always_off"
+            case traceIDRatio = "traceidratio"
+            case parentBasedAlwaysOn = "parentbased_always_on"
+            case parentBasedAlwaysOff = "parentbased_always_off"
+            case parentBasedTraceIDRatio = "parentbased_traceidratio"
+            case parentBasedJaegerRemote = "parentbased_jaeger_remote"
+            case jaegerRemote = "jaeger_remote"
+            case xray
+        }
+
+        package enum ArgumentBacking: Equatable, Sendable {
+            case traceIDRatio(samplingProbability: Double)
+            case jaegerRemote(endpoint: String, pollingInterval: Duration, initialSamplingRate: Double)
+        }
+
+        package var backing: Backing
+
+        package var argument: ArgumentBacking?
+
+        /// A sampler that always records the span.
+        public static let alwaysOn: Self = .init(backing: .alwaysOn)
+
+        /// A sampler that always drops the span.
+        public static let alwaysOff: Self = .init(backing: .alwaysOff)
+
+        /// A sampler that records a span based on ratio-based probability.
+        ///
+        /// - TODO: This is still in Development status in the OTel spec; should it be included in our 1.0 API?
```

Comment:
I would like to keep it as part of 1.0 because it was community-contributed. How about explicitly marking it as experimental and excluding it from sem-ver though? E.g. renaming it to `experimentalTraceIDRatio(ratio: Double = 1.0)`?